apps/api/src/deployment/services/top-up-custodial-deployments/top-up-custodial-deployments.service.ts

```diff
@@ -97,27 +99,37 @@ export class TopUpCustodialDeploymentsService implements DeploymentsRefiller {
       }
 
       balancesService.recordTx(amount, this.MIN_FEES_AVAILABLE);
-
-      await this.topUpDeployment(
-        {
-          dseq,
-          amount,
-          denom,
-          owner,
-          grantee
-        },
-        client,
-        options
-      );
-
-      hasTopUp = true;
-      summary.inc("deploymentTopUpCount");
-      summary.ensurePredictedClosedHeight(predictedClosedHeight);
+      const topUpParams = {
+        dseq,
+        amount,
+        denom,
+        owner,
+        grantee
+      };
+      const logParams = { params: { topUpParams, masterWallet: grantee }, dryRun: options.dryRun };
+
+      try {
```

Comment:
the issue was that without this error handler the loop would stop for the current wallet resulting into the rest not being topped up. same for the other service below